client/state/directly/reducer.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DIRECTLY_INITIALIZE
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const isInitialized = createReducer( false, {
+	[ DIRECTLY_INITIALIZE ]: () => true,
+} );
```

Comment:
there's probably no actual need to track an `isInitialized` state. it _might_ be the case but probably isn't. for one, we can represent the initialization state as the presence or absence of other data which we care about; second, do we need to directly show if it has initialized yet to the user?

I like to challenge it whenever I see these kinds of boolean state values.

further, we don't really have a need to make such heavy reducer functions when the logic is actually primitive:

```js
export config = ( state = null, { type, config } ) =>
	DIRECTY_INITIALIZE === type
		? config
		: state;
```